src/index.html

```diff
@@ -6,9 +6,30 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img src="images/logo.png" class="logo"
+        alt="moyo.png">
```

Comment:
'alt' should describe what is displayed on the image, like a text representation of it. Filename-like text is not the best choice.
I would put something like `Logo` or `Moyo logo` or `Moyo`